web/src/layouts/AppLayout.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link, Outlet } from 'react-router-dom'
+import { useState } from 'react'
+import { apiGet } from '../lib/api'
+import AccountModal from '../components/AccountModal'
+
+export default function AppLayout() {
```

Comment:
Right now, AppLayout handles:

- layout grid
- routing outlet
- navigation bar
- modal state
- user fetch logic

That’s five separate concerns (not SOLID). It should only have one: layout.
You can extract the “Account” control into its own component that knows how to fetch and display account info